src/AspNetCore/Src/applicationmanager.cxx

```diff
@@ -189,6 +189,31 @@ APPLICATION_MANAGER::GetApplicationInfo(
     return hr;
 }
 
+BOOL
+APPLICATION_MANAGER::FindConfigChangedApplication(
```

Comment:
I'd prefer to have this as a utility function as it doesn't depend on anything else in the application manager.

Reply:
this method should be static. However, it depends on config and logically associated with a hashttable. Put is to into utility is not good for maintenance.